src/back/bootstrap/index.js

```diff
@@ -9,10 +9,18 @@ const
 
 module.exports = (app) => {
     db(formatMongooseDbUriFromEnv(env));
+    extendRequest();
     preMiddleware(app);
     return app.use(controllers);
 };
 
+function extendRequest() {
+    const authService = require('../controllers/auth/auth.service'),
+        http = require('http');
+    http.IncomingMessage.prototype.getCurrentUser = authService.getCurrentUser;
+    http.IncomingMessage.prototype.isAuthenticated = authService.isAuthenticated;
```

Comment:
I couldn't find methods `getCurrentUser` and `isAuthenticated` used anywhere. Could you advice where they are used?